.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
I even looked at the official README of node-coverage, and there are also saying that the [token is needed](https://github.com/nickmerwin/node-coveralls#tap):

![image](https://cloud.githubusercontent.com/assets/11187343/22622567/80191e0a-eaf2-11e6-92ab-d3cce197ce5c.png)

I'm fairly certain that the token is still needed.